examples/models/llama/runner/native.py

```diff
@@ -89,7 +101,6 @@ def build_args_parser() -> argparse.ArgumentParser:
     parser.add_argument(
         "-kv",
         "--kv_cache",
```

Comment:
I think we'd want the default to still be True?

Reply:
Oh yeah this was weird since "store_true" works by having a default of False, but the default is here set to True so it's just always True regardless of what you put